webapp/layouts/default.vue

```diff
@@ -193,6 +196,16 @@ export default {
         return this.$route.path === url
       }
       return this.$route.path.indexOf(url) === 0
+    },
+    // TODO method this is a duplicate from /pages/profile/_id/_slug.vue
+    // where to put this?
```

Comment:
@ulfgebhardt if you want to remove redundancy, VueJS has a couple of ways to do this. My suggestion would be a mixin. https://vuejs.org/v2/guide/mixins.html

Reply:
Implemented as Mixin